packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -59,7 +59,7 @@ const chatMessageStyles: ComponentSlotStylesInput<
     // Otherwise, if the variable is not provided, the default appearance logic will be used for actions menu.
     ...(_.isNil(v.showActionMenu) && {
       ':hover': {
-        [`& .${ChatMessage.slotClassNames.actionMenu}`]: {
+        [`> .${ChatMessage.slotClassNames.actionMenu}`]: {
```

Comment:
that's an important fix for actions menu, otherwise, when hovering one message, actions menu is displayed for all messages